kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -5,6 +5,7 @@
 
 # Other imports
 import os
```

Comment:
```suggestion
```

everything can be done with `Path`

Reply:
I don't see how `os.path.split(self.file_name)` can be done without `os`. [There is no equivalent](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module) for this operation.